web/static/css/app.css

```diff
@@ -94,6 +113,22 @@ h2 > span.b {
   background-repeat: no-repeat;
 }
 
+.filter-check:not(:checked)~.filter-sidebar {
+  margin-left: -80vw;
+  -webkit-transition-duration: 0.5s;
+	-moz-transition-duration: 0.5s;
+	-o-transition-duration: 0.5s;
+	transition-duration: 0.5s;
+}
+
+.filter-check:checked~.filter-sidebar {
+  margin-left: 0;
+  -webkit-transition-duration: 0.5s;
+	-moz-transition-duration: 0.5s;
+	-o-transition-duration: 0.5s;
+	transition-duration: 0.5s;
+}
+
```

Comment:
I've found drying out this type of css can make it a little more maintainable, something like this:
```css
.filter-check:not(:checked)~.filter-sidebar, .filter-check:checked~.filter-sidebar {
  -webkit-transition-duration: 0.5s;
	-moz-transition-duration: 0.5s;
	-o-transition-duration: 0.5s;
	transition-duration: 0.5s;
}

.filter-check:not(:checked)~.filter-sidebar {
  margin-left: -80vw;
}

.filter-check:checked~.filter-sidebar {
  margin-left: 0;
}
```